src/gdgem/srs/ParserVMM3TestData.h

```diff
@@ -29,7 +29,7 @@ std::vector<uint8_t> marker_3_vmm1_3 {
   0x00, 0x00, 0x00, 0x00,             // Offset overflow last frame
   0x00, 0x00, 0x00, 0x00, 0x04, 0x01, // marker 1: vmm1, timeStamp 1
   0x00, 0x00, 0x00, 0x00, 0x08, 0x02, // marker 2: vmm2, timeStamp 2
-  0x00, 0x00, 0x00, 0x00, 0x0c, 0x03, // marker 3: vmm3, timeStamp 3
```

Comment:
clang unhappy about this? it has been allowed forever

Reply:
Are you referring to the comma at the end? I did not get any errors, but thought that the comma is not necessary.

Reply:
Ahh, programmer's OCD ;-) I have no problem with this.